recipes/libepoxy/all/conanfile.py

```diff
@@ -0,0 +1,104 @@
+from conans import ConanFile, Meson, tools
+import os
+import shutil
+import glob
+
+
+class EpoxyConan(ConanFile):
+    name = "libepoxy"
+    description = "libepoxy is a library for handling OpenGL function pointer management"
+    topics = ("conan", "libepoxy", "opengl")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/anholt/libepoxy"
+    license = "MIT"
+    generators = "pkg_config"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "glx": [True, False],
+        "egl": [True, False],
+        "x11": [True, False]
+    }
+    default_options = {
+        "shared": True,
```

Comment:
Unless this package is an exception (as I did with cpython)
```suggestion
        "shared": False,
```

Reply:
The thing is that the upstream project is built as a shared library by default: https://mesonbuild.com/Builtin-options.html#core-options
Changing the default means that we expose a LOT of consumers to situation which are not really tested. This caused issues in several other projects (eg glib).
Regarding this specific package, ubuntu and debian provide only the shared library: https://packages.debian.org/stretch/amd64/libepoxy-dev/filelist https://packages.ubuntu.com/eoan/amd64/libepoxy-dev/filelist

Reply:
Maybe add a comment why because I envision future pull requests changing this.